### PR TITLE
Ft assign drivers to buses

### PR DESCRIPTION
**Pull request by fab-ryan**
## What does this PR do?
The Operator will be able to assign registered drivers to registered buses 
## How should this be tested?
- clone this [repo](https://github.com/atlp-rwanda/Dominators-Phantom-FE/pull/24)
- checkout to branch `ft-assign-drivers-to-buses`
- then open command prompt CLI to install dependencies by using `npm i` or `npm install` but for those who uses **NPM**  but for those who uses **yarn**. just type `yarn`.
- start Serve with any port you wish
- by using Base URL plus `assign` but this URL will be accessed by Operator only
- once new driver have been given new driver has been assigned to bus or un-assigned they will notification to his/her email 

## what to expect 
- notification to email for assigned and unassigned driver
- error messages
- successful messages displayed

### CODE REVIEW

**Review by GilbertUp — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

File: crude-route.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "id": 5,
+      "from": "KIMIRONKO",
+      "to": "KABUGA",
+      "code": "300",
+      "distance": "4km"
+    }
+  ]
+}
```
Comment (Anguandia) [inline crude-route.json]: We can not yet get this from the backend?

File: src/utils/db.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const db = "http://localhost:4000";
```
Comment (Anguandia) [inline src/utils/db.js]: I think it will offer greater flexibility if we read the backend url from the environment.

**Review by Anguandia — CHANGES_REQUESTED** [review]
Great work, please look into the following:
- Please describe how I should test this PR locally (on my laptop)
- I can only see the home and routes pages, is `ft-assign-drivers-to-buses` a misnomer?
- Additionally, please find time to respond to the in-code comments

**Review by furebo — CHANGES_REQUESTED** [review]
Please rebase and resolve the conflicts.

**Review by furebo — APPROVED** [review]

**Pull request merged**